test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -278,11 +354,10 @@ void static command_line_arg_check(int argc, char *argv[],
                     " + warmup * sizes (8/4KB) <= max length \n"
                     "[-r number of nodes at target, use only with -t] \n"
                     "[-l number of nodes at initiator, use only with -t, "
-                    "l/r cannot be used together] \n");
+                    "l/r cannot be used together] \n"
+                    "[-C thread-safety-config: SINGLE, FUNNELED, SERIALIZED, or MULTIPLE] \n"
+                    "[-T num-threads] \n");
         }
-#ifndef VERSION_1_0
-        shmem_finalize();
```

Comment:
what's the reason for taking out finalize?

Reply:
This came from one of the initial patches. Will dig up to find any reason.